server/sonar-server/src/test/java/org/sonar/server/qualityprofile/RuleActivatorMediumTest.java

```diff
@@ -1059,6 +1059,9 @@ public void bulk_change_severity() {
     verifyHasActiveRuleInDbAndIndex(ActiveRuleKey.of(XOO_P2_KEY, XOO_X2), BLOCKER, INHERITED, Collections.<String, String>emptyMap());
     verifyHasActiveRuleInDbAndIndex(ActiveRuleKey.of(XOO_P3_KEY, XOO_X1), BLOCKER, INHERITED, ImmutableMap.of("max", "7"));
     verifyHasActiveRuleInDbAndIndex(ActiveRuleKey.of(XOO_P3_KEY, XOO_X2), BLOCKER, INHERITED, Collections.<String, String>emptyMap());
+    assertProfileHasBeenUpdatedAutomatically(XOO_P1_KEY);
```

Comment:
Why in bulk change the profile is not considered as having been updated manually ?